target/scripts/helpers/utils.sh

```diff
@@ -4,6 +4,15 @@ function _escape() {
   echo "${1//./\\.}"
 }
 
+# Replaces a string so that it can be used inside
+# `sed` safely.
+#
+# @param ${1} = string to escape
+# @output     = prints the escaped string
+function _escape_for_sed() {
+  sed -E 's/[]\/$*.^[]/\\&/g' <<< "${1:?String to escape for sed is required}"
+}
```

Comment:
Likewise, a similar function exists in `db.sh`:

https://github.com/docker-mailserver/docker-mailserver/blob/5bada0a83b0875b36934e1de856b724979c83346/target/scripts/helpers/database/db.sh#L166-L173

I am not familiar enough with the two to compare them. I barely recall how I came up with mine here at the time 😅 

That said, I'd be a bit cautious replacing / changing the `db.sh` method, if I avoided escaping certain characters with the method there, it may have been for good reason, but unfortunately is lacking in test coverage AFAIK.

Personally my future plans were to write a rust CLI tool instead of dealing with bash scripts where alot of the issues related to this was from. But that would be a larger effort to pursue :(

Reply:
There really is a difference: mine seems to be more comprehensive

```console
$ sed 's/\([/&]\)/\\\1/g' <<< '$\\'
$\\
$ sed -E 's/[]\/$*.^[]/\\&/g' <<< '$\\'
\$\\\\
```

Doesn't matter though, as I will not be fiddling with `db.sh` anyways :D No time 🚀

Reply:
Might be worth adding a comment to the `db.sh` one referencing the newer helper you've contributed.

I would be open to switching to yours, I'm just overly cautious about a change to `db.sh` messing something up 😅 That is code I do not enjoy troubleshooting.